cdk-s3-lambda-textract-comprehend-s3/example-pattern.json

```diff
@@ -0,0 +1,52 @@
+{
+  "title": "Serverless Document Processing and Identification using Textract and Comprehend with CDK",
+  "description": "To analyse and identify the uploaded document whether it is a valid Government ID. Implemented in CDK",
+  "language": "Python",
+  "level": "200",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This sample project demonstrates how to analyze and identify the uploaded document as a valid government issued ID. Implemented with CDK (Python)"
```

Comment:
This is a copy of the description, @paulkannan can you give us more details on how the pattern works? Nothing too much, but dive deeper into the pattern slighty.